dftimewolf/lib/collectors/bigquery.py

```diff
@@ -25,36 +25,37 @@ def __init__(self,
     self._project_name = ""
     self._query = ""
     self._description = ""
+    self._panda_output = False
```

Comment:
pandas? (and similarly throughout the file...)

Reply:
Done.